.gitignore

```diff
@@ -3,3 +3,4 @@
 /target/
 Cargo.lock
 runtime/**/target
+.vscode
```

Comment:
Why add this folder? I remember some debug configuration file can be put here.

Reply:
it just vscode configuration file. it may be contain some private info, ignore it looks better.